lms/djangoapps/courseware/access.py

```diff
@@ -848,8 +851,14 @@ def _has_catalog_visibility(course, visibility_type):
 def _is_descriptor_mobile_available(descriptor):
     """
     Returns if descriptor is available on mobile.
+
+    If the ignore_mobile_available_flag is enabled, the mobile_available flag
+    is ignored and access will be granted.
     """
-    return ACCESS_GRANTED if descriptor.mobile_available else MobileAvailabilityError()
+    if descriptor.mobile_available or IgnoreMobileAvailableFlagConfig.ignore_mobile_available_flag():
```

Comment:
Nit: The method name, `ignore_mobile_available_flag`, seems redundant with the class name `IgnoreMobileAvailableFlagConfig`.  See below about using the standard `enabled` field instead.